cmd/kubeadm/app/cmd/phases/reset/unmount.go

```diff
@@ -0,0 +1,29 @@
+// +build !linux
```

Comment:
Let's not do premature platform splitting in reset at this point. It's almost all Linux dependent at this point and we need to have the whole picture in mind to take the best approach.

Reply:
but this function certainly should not run on _windows.go (NO-OP).

Reply:
@rosti I guess  I don't understand, if I don't have this file the code will not pass checks because it does not have the functions used in the `unmount_linux.go` file on windows and mac. For both windows and mac this is a no-op. How are you wanting me to handle compiling for windows and linux if not with this file?

Reply:
Ok, we can leave it like this for now, but we may need to redesign and rename the files.
The question is if we should place `absoluteKubeletRunDirectory` in the platform specific portions of `cleanupnode.go`? So the idea is to have `cleanupnode_unix.go` and `cleanupnode_windows.go` with `absoluteKubeletRunDirectory` and a few other things in it.

But, certainly, that must be done in another PR.